modules/virt-editing-vmis-web.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/cnv-manage-vmis.adoc
+
+[id="virt-editing-vmis-web_{context}"]
+= Editing a standalone virtual machine instance using the web console
+
+You can edit annotations and labels for a standalone virtual machine instance (VMI) using the web console. Other items displayed in the *Details* page for a standalone VMI are not editable.
+
+.Procedure
+
+. Click *Workloads* -> *Virtualization* from the side menu. A list of virtual machines (VMs) and standalone VMIs display.
```

Comment:
The qualifying subject as I see it here is the one closest to the verb - in this case, it is "standalone VMIs", not the singular noun "list," therefore we have the singular verb "display."
Please advise if you see the qualifying subject here as being "list," and in that case the plural "displays" would apply in this subject-verb agreement.

Reply:
In this subject-verb agreement, the qualifying subject is the one closest to the verb, therefore I used the verb "display" in singular voice.

Reply:
It would be great to have access to the Chicago Manual of Style but that is a paid subscription, not sure if CCS provides access to it. Therefore, I checked with Ingrid for editing consultation and she said:
I would use "displays", because the subject of the sentence is list. "virtual machines (VMs)" and "standalone VMIs" are both objects of the preposition "of"."

Changing to "displays"